File: Day51/ConsumeService/Models/Product.cs
using System.ComponentModel.DataAnnotations;

namespace ConsumeService.Models
{
    public class Product
    {
        public int Id { get; set; }
      
        public string Name { get; set; }
      
        public int Quantity { get; set; }
        
        public float Price { get; set; }
    }
}

File: Day78/productapp/src/components/Product.js
import React from "react";
import './Product.css';

function Product(props){
    return (
        <div className="Product alert alert-info">
                     Product Id:{props.product.id}
            <br/>
            Product Name:{props.product.name}
            <br/>
            Product Price:{props.product.price}
            <br/>
            Product Quantity:{props.product.quantity}
            <br/>
        </div>
    )
}

export default Product;